docs/source/_toctree.yml

```
@@ -296,6 +296,8 @@
       title: RemBERT
     - local: model_doc/resnet
       title: ResNet
+    - local: model_doc/resnetd
+      title: ResNetD
```

Will need to be added to the main README as well. Also it's ResNet-D in the paper:
```suggestion
      title: ResNet-D
```